pkg/network/server_test.go

```diff
@@ -392,6 +392,7 @@ func startWithCleanup(t *testing.T, s *Server) {
 	go s.Start()
 	t.Cleanup(func() {
 		s.Shutdown()
+		_ = s.log.Sync()
```

Comment:
We don't need it in test then since `s.Shutdown()` performs logs sync.

Reply:
fixed